lua/hbac/command.lua

```diff
@@ -1,26 +1,38 @@
 local state = require("hbac.state")
+local autocommands = require("hbac.autocommands")
 
 local M = {
-	subcommands = {}
+	subcommands = {},
 }
 
 M.subcommands.close_unpinned = function()
 	local curbufnr = vim.api.nvim_get_current_buf()
 	local buflist = vim.api.nvim_list_bufs()
 	for _, bufnr in ipairs(buflist) do
 		if vim.bo[bufnr].buflisted and bufnr ~= curbufnr and not state.is_pinned(bufnr) then
-			vim.cmd('bd ' .. tostring(bufnr))
+			require("hbac.setup").opts.close_command(bufnr)
 		end
 	end
 	print("Hbac: Closed unpinned buffers")
 end
 
-
 M.subcommands.toggle_pin = function()
 	local bufnr = vim.api.nvim_get_current_buf()
 	print("Hbac: Buffer " .. bufnr .. " " .. state.toggle_pin(bufnr))
 end
 
+M.subcommands.toggle_autoclose = function()
+	if state.autoclose_enabled then
+		state.autoclose_enabled = false
+		autocommands.autoclose.disable()
+		print("Hbac: Autoclose disabled")
```

Comment:
in this case it makes sense to check if pcall returns true and set the state after
```suggestion
		if autocommands.autoclose.disable() then
		    state.autoclose_enabled = false
		    print("Hbac: Autoclose disabled")
		end
```

Reply:
Couldn't the pcall failure indicate that the augroup didn't exist/was already removed by an external actor?

Reply:
Good point. That would be an implicit assumption though. Lets add a comment then.

Reply:
Absolutely, good idea.